<launch>
  <node pkg="video_writer_py" name="video_writer_fisheye_py" type="video_writer_py.py" output="screen">
  <!--param name="subscribed_topic" type="string" value="/dji_sdk/image_raw"/-->
  <!--param name="subscribed_topic" type="string" value="/my_video"/-->
  <param name="subscribed_topic" type="string" value="/fisheye/image"/>
  
  <param name="show_video_flag" type="bool" value="false"/>
  <param name="save_video_flag" type="bool" value="true"/>
  <param name="video_file_name" type="string" value="/home/ubuntu/2017/src/video_writer_py/video/out_fisheye.avi"/>
  <param name="rate" type="double" value="30.0"/>
  </node>
</launch>
